### PR TITLE
Simplify release script and skip existing releases

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
We simplified the script and utilising better the `chart-releaser` CLI. Now we iterate through all charts and we package them but we will upload only the packages are new. Added a new flag for `chart-releaser` which will help us to skip the existing packages.

#### Ticket Link
https://mattermost.atlassian.net/browse/CLD-3970

### CODE REVIEW

**Comment by phoinixgrr** [comment]
Great work here. 

I'm always in font of simplifying things 👍 
Shellcheck comes green, no objections merging this

**Review by phoinixgrr — APPROVED** [review]

**Comment by spirosoik** [comment]
@pfltdv I would like your feedback first before I merge. I removed the complexity and I just followed what everyone does out there actually for the public helm charts (deliveryhero, grafana, prometheus etc.)

File: scripts/release.sh [hunk 1]
```diff
@@ -82,7 +48,7 @@ release_charts() {
         -w /src \
         -e CR_TOKEN="${CR_TOKEN}" \
         "${DOCKER_IMAGE_CR}" \
-        upload -o mattermost -r mattermost-helm
+        upload --skip-existing -o mattermost -r mattermost-helm
```
Comment (pfltdv) [inline scripts/release.sh:51]: :heart: I love this flag. Great work Spiros.

**Review by pfltdv — APPROVED** [review]

**Comment by pfltdv** [comment]
> @pfltdv I would like your feedback first before I merge. I removed the complexity and I just followed what everyone does out there actually for the public helm charts (deliveryhero, grafana, prometheus etc.)

Amazin work @spirosoik . It is so simple now with only adding one flag.

**Pull request merged**